create_destroy_db.sql

```diff
@@ -0,0 +1,94 @@
+-- MySQL Script generated by MySQL Workbench
```

Comment:
@GWhite24 you have to run this file first before you start the app or the db won't be in place to connect to